subprojects/ivy/src/main/java/org/gradle/api/publish/ivy/internal/publisher/IvyDescriptorFileGenerator.java

```diff
@@ -39,17 +41,28 @@
 import java.io.Writer;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
 public class IvyDescriptorFileGenerator {
     private static final String IVY_FILE_ENCODING = "UTF-8";
     private static final String IVY_DATE_PATTERN = "yyyyMMddHHmmss";
+    private static final Action<XmlProvider> ADD_GRADLE_METADATA_MARKER = new Action<XmlProvider>() {
```

Comment:
Should that go as a field of `MetadataParser` as well?
Feels strange to have the Maven related code reaching into Ivy related code otherwise.

Reply:
Same constant, different classes, irrelevant comment ...

Reply:
:chains: